examples/advanced/create-react-app/README.md

```diff
@@ -14,7 +14,7 @@ This is what you should see:
 
 <img alt="screenshot of the React application" src="README.media/react-app-screenshot.png" />
 
-If instead you want to build just the tokens you can run `npm run build-dictionary`. This command will generate the files in the `src/style-dictionary-dist` folder, inside the CRA source folder.
+If instead you want to build the tokens you can run `npm run build-dictionary`. This command will generate the files in the `src/style-dictionary-dist` folder, inside the CRA source folder.
```

Comment:
need to modify this to say we are only building tokens and not running the react app to differentiate it from above